moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -48,6 +48,11 @@
 
 #include <boost/assert.hpp>
 
+/* Terminology
+ *   Model Frame: the root frame of the URDF model
```

Comment:
```suggestion
 *   Model Frame: RobotModel's model frame == PlanningScene's planning frame
     If the SRDF defines a virtual, non-fixed (e.g. floating) joint, this is the parent of this virtual joint.
     Otherwise, it is the root joint of the URDF model.
```
Compare the existing documentation in RobotModel:
https://github.com/ros-planning/moveit/blob/3a57141b48435de0a6026f9fc3496165c72e19e6/moveit_core/robot_model/include/moveit/robot_model/robot_model.h#L455-L457

https://github.com/ros-planning/moveit/blob/3a57141b48435de0a6026f9fc3496165c72e19e6/moveit_core/robot_model/src/robot_model.cpp#L928-L931